Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -631,7 +631,15 @@ variable {b : ℝ} {f : ℕ → ℝ} {z : ℕ → E}
 /-- **Dirichlet's Test** for monotone sequences. -/
 theorem Monotone.cauchySeq_series_mul_of_tendsto_zero_of_bounded (hfa : Monotone f)
     (hf0 : Tendsto f atTop (𝓝 0)) (hgb : ∀ n, ‖∑ i in range n, z i‖ ≤ b) :
-    CauchySeq fun n ↦ ∑ i in range (n + 1), f i • z i := by
+    CauchySeq fun n ↦ ∑ i in range n, f i • z i := by
+  suffices CauchySeq fun n ↦ ∑ i in range (n + 1), f i • z i by
```

Comment:
This should be easier using [CauchySeq.comp_tendsto](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/UniformSpace/Cauchy.html#CauchySeq.comp_tendsto).

Anyway, I would assume that it should be possible to modify the previous proof to give the result directly (but I didn't check myself).

Reply:
Maybe not...

To go via [CauchySeq.comp_tendsto](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/UniformSpace/Cauchy.html#CauchySeq.comp_tendsto), one would need a congruence lemma that says that when two sequences agree eventually, one is Cauchy iff the other one is. But I can't find this in Mathlib.

Reply:
(The problem is that `n - 1 + 1 = n` does not hold for all `n`.)

Reply:
So I would suggest that you add a general statement
```lean
lemma CauchySeq.shift {α : Type*} [uniformSpace : UniformSpace α]  (u : ℕ → α) (k : ℕ) :+1: 
    CauchySeq (fun n ↦ u (n + k)) ↔ CauchySeq u := ...
```
say in `Mathlib.Topology.UniformSpace.Cauchy` and use that in the proof.

Reply:
Done.